docs/production-deployment/cloud/nexus/index.mdx

```diff
@@ -0,0 +1,87 @@
+---
+id: index
+slug: /cloud/nexus
+title: Temporal Nexus
+description: Discover Temporal Nexus, a powerful feature for connecting durable executions across team, namespace, region, and cloud boundaries. Nexus also enables each team to have their own namespace for improved security, troubleshooting, and blast radius isolation.
+sidebar_label: Temporal Nexus 
+tags:
+  - temporal cloud
+  - temporal nexus
+keywords:
+  - Temporal Nexus
+  - cross-namespace workflows
+  - modular workflow design
+  - workflow orchestration
+  - Temporal Cloud security
+  - Temporal Cloud services
+  - distributed workflows
+  - workflow API management
+---
+
+:::tip SUPPORT, STABILITY, and DEPENDENCY INFO
+
+Temporal Nexus is in [Public Preview](https://docs.temporal.io/evaluate/development-production-features/release-stages\#public-preview) for Temporal Cloud.
+
+:::
+
+This Temporal Nexus guide covers the following topics:
+
+- [Introduction to Nexus](#nexus-introduction)  
+- [Using Nexus](/cloud/nexus/using-nexus)  
+- [Pricing](/cloud/nexus/pricing)  
+- [Getting Started](/cloud/nexus/getting-started)  
+- [Operations](/cloud/nexus/operations)  
+
+
+## Introduction to Nexus {#nexus-introduction}
+Temporal Nexus is a feature of the Temporal platform designed to connect durable executions across team, Namespace, region, and cloud boundaries. 
+It promotes a more modular architecture for sharing a subset of your team’s capabilities through well-defined service API contracts for other teams to use, that abstract underlying Temporal primitives, like Workflows, or execute arbitrary code.
```

Comment:
```suggestion
It promotes modular architectures that share your team’s capabilities through well-defined service API contracts for other teams to use.
It abstracts underlying Temporal primitives, like Workflows and Activities, or executes arbitrary code.
```

Reply:
Yeah this sentence is difficult to read. There eventually will need to be a new intro since we're typically trying to stay away from terms like primitives.